src/libraries/Microsoft.Extensions.Logging/gen/LoggerMessageGenerator.Emitter.cs

```diff
@@ -443,63 +445,56 @@ static string GetLogLevel(LoggerMethod lm)
                 }
             }
 
-            private void GenEnumerationHelper(LoggerClass lc)
```

Comment:
The enumeration helper is either generated (only once), or not at all into `__LoggerMessageGenerator`